examples/data_pipeline/pipelines.py

```diff
@@ -8,10 +8,8 @@
 from torchtext.experimental.transforms import (
     basic_english_normalize,
     TextSequentialTransforms,
-    VocabTransform,
-    VectorTransform,
```

Comment:
Why are we getting rid of the transforms? Is this because these building blocks have their own `forward` method defined?

Reply:
Switch to the `forward` func. It works for a list of tokens on JIT mode.